<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustWindow</class>
 <widget class="QMainWindow" name="CustWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户详单</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="container">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QueryBox" name="queryBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="nameLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>39</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="nameEdit">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="genCombo">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>10</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>性别</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="posLabel">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>10</y>
            <width>39</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>职务：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="posEdit">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="mainPepLabel">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>10</y>
            <width>51</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>维护人：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="mainPepEdit">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="areaLabel">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>10</y>
            <width>51</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>区域：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="areaEdit">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="mainLevLabel">
          <property name="geometry">
           <rect>
            <x>770</x>
            <y>10</y>
            <width>61</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>维护等级：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="mainLevEdit">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="queryButton">
          <property name="geometry">
           <rect>
            <x>1050</x>
            <y>10</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="cateEdit">
          <property name="geometry">
           <rect>
            <x>960</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="cateLabel">
          <property name="geometry">
           <rect>
            <x>920</x>
            <y>10</y>
            <width>61</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>类别：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="specEdit">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="specLabel">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>10</y>
            <width>39</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>专业：</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="detailBox">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>详情</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="custTable">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="insertRowAction">
   <property name="text">
    <string>&amp;插入</string>
   </property>
   <property name="toolTip">
    <string>插入新行</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>&amp;删除</string>
   </property>
   <property name="toolTip">
    <string>删除该行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QueryBox</class>
   <extends>QGroupBox</extends>
   <header location="global">querybox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>custTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
